### PR TITLE
ES6 Upgrade cleanup

### PR DESCRIPTION
**Pull request by sujaykathrotia**
- Removed unnecessary Home and About import from Application component
- Changed normal function callback to Arrow functions
- Changed require statements to import statements
- Changed module.export to export default

### CODE REVIEW

File: app/templates/stores/ApplicationStore.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ class ApplicationStore extends BaseStore {
     }
     handlePageTitle(currentRoute) {
         var self = this;
-        this.dispatcher.waitFor(RouteStore, function () {
+        this.dispatcher.waitFor(RouteStore, () => {
```
Comment (redonkulus) [inline app/templates/stores/ApplicationStore.js]: probably don't need the `self` variable anymore right?

Comment (sujaykathrotia) [inline app/templates/stores/ApplicationStore.js]: Yes. I forgot to change it.

**Comment by mridgway** [comment]
Nice. Thanks!

**Comment by yahoocla** [comment]
CLA is valid!

**Pull request merged**